spec/features/visitor_creates_an_account_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+describe User, type: :feature do 
+  describe "A visitor" do 
+    it "can register as a user with correct field inputs" do 
+      visit root_path
+      click_on "Create Account"
+      expect(current_path).to eq(new_user_path)
+
+      fill_in "First name", with: "First Name"
+      fill_in "Last name", with: "Last Name"
+      fill_in "Email", with: "Email"
+      fill_in "Phone number", with: "Phone"
+      fill_in "Password", with: "Password"
+      
+      click_on "Sign Up"
+
+      expect(page).not_to have_content("Create Account")
```

Comment:
@njgheorghita what would be our expected path here?  Or, where should the user be right after creating an account? Dashboard?  Profile?